packages/liveblocks-react/src/liveblocks.tsx

```diff
@@ -1114,6 +1117,60 @@ function useDeleteAiChat() {
   );
 }
 
+/**
+ * Returns a function to send a message in an AI chat.
+ *
+ * @example
+ * const sendMessage = useSendAiMessage(chatId);
+ * sendMessage("Hello, Liveblocks AI!");
+ */
+function useSendAiMessage(
+  chatId: string,
+  options?: { copilotId?: string }
+): (message: string) => void {
+  const client = useClient();
+  const copilotId = options?.copilotId;
+
+  return useCallback(
+    (message: string) => {
+      const messages = client[kInternal].ai.signals
+        .getChatMessagesForBranchΣ(chatId)
+        .get();
+
+      const lastMessageId = messages[messages.length - 1]?.id ?? null;
+
+      const content = [{ type: "text" as const, text: message }];
+      const newMessageId = client[kInternal].ai[
+        kInternal
+      ].context.messagesStore.createOptimistically(
+        chatId,
+        "user",
+        lastMessageId,
+        content
+      );
+
+      const targetMessageId = client[kInternal].ai[
+        kInternal
+      ].context.messagesStore.createOptimistically(
+        chatId,
+        "assistant",
+        newMessageId
+      );
+
+      void client[kInternal].ai.askUserMessageInChat(
+        chatId,
+        { id: newMessageId, parentMessageId: lastMessageId, content },
+        targetMessageId,
+        {
+          stream: false,
+          copilotId: copilotId as CopilotId | undefined,
+        }
+      );
```

Comment:
There's quite some duplication here with what we have in `AiChatComposer` in the other package too. I would love to see if we could instead expose this as a private method in core, so we can avoid having to expose the `messagesStore` and duplicate the optimistic message creation, and calling `askUserMessageInChat`. Fine for now, but something to clean up soon.